tarofreighter/chain_porter.go

```diff
@@ -3,6 +3,7 @@ package tarofreighter
 import (
 	"bytes"
 	"fmt"
+	"math/rand"
```

Comment:
Should use `crypto/rand` to silence the linter.

Reply:
Using backoff now instead. I don't think the rand aspect is necessary anymore.